### PR TITLE
Adding ib-insync, eventkit, and nest-asyncio

### PR DESCRIPTION
**Pull request by yehoshuadimarsky**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by yehoshuadimarsky** [comment]
@conda-forge/help-python

File: recipes/eventkit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python>=3.6
+    - pip
+    - setuptools
+  run:
+    - python>=3.6
```
[the remark this answers is not in the document]
Comment (yehoshuadimarsky) [reply recipes/eventkit/meta.yaml]: I don't understand - a space where?

File: recipes/nest-asyncio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python>=3.5
+    - pip
+    - setuptools
+  run:
+    - python>=3.5
```
Comment (xhochy) [inline recipes/nest-asyncio/meta.yaml]:
```suggestion
    - python >=3.5
```

File: recipes/nest-asyncio/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: bd1cb7df2ea979e57d8ad02493ad85f9afbf1fcea3dfe34239da8c0dda98087e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python>=3.5
```
Comment (xhochy) [inline recipes/nest-asyncio/meta.yaml]:
```suggestion
    - python >=3.5
```

File: recipes/eventkit/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (xhochy) [reply recipes/eventkit/meta.yaml]: I made suggested changes to outline where

[the remark this answers is not in the document]
Comment (yehoshuadimarsky) [reply recipes/eventkit/meta.yaml]: thanks - now I see what you mean.

**Pull request merged**